test/e2e/specs/test.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
 
   'test header': function (browser) {
     browser
+      .pause(90000)
```

Comment:
90 seconds is a lot, is this needed?, why can't we wait till the element is visible instead of sleeping for 90 seconds